lib/livebook/config.ex

```diff
@@ -63,7 +63,11 @@ defmodule Livebook.Config do
   """
   @spec data_path() :: String.t()
   def data_path() do
-    Application.get_env(:livebook, :data_path) || :filename.basedir(:user_data, "livebook")
+    if data_path = Application.get_env(:livebook, :data_path) do
+      Path.join(data_path, "livebook")
```

Comment:
Any specific reason for this? Currently we have:

```
      --data-path          The directory to store Livebook configuration,
                           defaults to "livebook" under the default user data directory
```

Reply:
Both nerves-livebook and fly have set it to `/data`, so I think the automatic nesting would be helpful. But we can removed it too. Currently we just write the "autosaved" folder to it.

Reply:
Oh, I see! I still wouldn't automatically nest it, so that the user has full control, but no strong opinion.